airflow/providers/google/cloud/sensors/dataflow.py

```diff
@@ -151,12 +205,14 @@ def __init__(
         self,
         *,
         job_id: str,
-        callback: Callable[[dict], bool],
+        callback: Callable | None = None,
```

Comment:
May I know why are we removing the arg annotation of `Callable` here? Is it no longer the case?

Reply:
Got it. If it can be arbitrary Callable, this change makes sense. Thanks!